.github/workflows/ci.yml

```diff
@@ -1,8 +1,10 @@
-name: C/C++ CI
+name: C/C++ patch CI
```

Comment:
@marshal-dteach Could you please explain why a "patch" was added in this context?

Reply:
It was added to have a descriptive CI name
![Capture](https://github.com/25077667/XORList/assets/36204988/07e251cc-3eef-43e3-9b98-f461db4e9fbf)